MedallionShell.Tests/GeneralTest.cs

```diff
@@ -350,7 +350,7 @@ public void TestVersioning()
             var version = typeof(Command).GetTypeInfo().Assembly.GetName().Version.ToString();
             var informationalVersion = (AssemblyInformationalVersionAttribute)typeof(Command).GetTypeInfo().Assembly.GetCustomAttribute(typeof(AssemblyInformationalVersionAttribute));
             Assert.IsNotNull(informationalVersion);
-            version.ShouldEqual(Regex.Replace(informationalVersion.InformationalVersion, "-.*$", string.Empty) + ".0");
+            version.ShouldEqual(Regex.Replace(informationalVersion.InformationalVersion, "[+-].*$", string.Empty) + ".0");
```

Comment:
I believe I needed this change because `informationalVersion.InformationalVersion` was evaluating to `1.6.2+87316dfa0e811d380d284c384a1bf525832ede7e.0` (as in [here](https://github.com/madelson/MedallionShell/pull/74#issuecomment-672810085)).

Reply:
Why does implementing this require changing the informational version?

Reply:
I didn't track down where exactly the diff was coming from, but https://github.com/madelson/MedallionShell/compare/master...release-1.7#diff-b0c1fafbf0a3bf095292c16e4f0de4f62b583981c35faf393170816b5183964dR21 already has this change, so we should be good here.

Reply:
@madelson I just figured out why! From https://learn.microsoft.com/en-us/dotnet/core/compatibility/sdk/8.0/source-link:

> As a side effect, commit information is included in the `InformationalVersion` value of built libraries and applications.